lib/version-check.js

```diff
@@ -77,9 +90,38 @@ module.exports = function (gulp, gutil) {
     validate(server.serverUrl(), server.pluginToken(), opts, cb, errorCallback);
   }
 
+  function checkVersionForThemeCheck(server, opts, cb, errorCallback) {
+    if (server.serverUrl() === undefined) {
+      var errMessage = 'A server URL is required in your configuration. ';
+      callbackOrThrowError(errorCallback, errMessage);
+    }
+    validate(server.serverUrl(), server.pluginToken(), opts, function() {
+      var localSkins = fs.readdirSync(skinsBaseDir).forEach(function(file) {
+        var skinDir = path.resolve(skinsBaseDir, file);
+        var skinPropFilePath = path.resolve(skinDir, "skin.properties");
+        if (isFile(skinPropFilePath)) {
+          var fileVal = fs.readFileSync(skinPropFilePath).toString();
```

Comment:
You probably want some sort of try/catch around any sort of file read operation.